<ui version="4.0" >
 <class>uiWindowTrustedRootCerts</class>
 <widget class="QDialog" name="uiWindowTrustedRootCerts" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>760</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>760</width>
    <height>2048</height>
   </size>
  </property>
  <property name="font" >
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle" >
   <string>Select a Trusted Root Server Certificate - XSupplicant</string>
  </property>
  <property name="windowIcon" >
   <iconset>icons/prod_color.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <property name="horizontalSpacing" >
    <number>0</number>
   </property>
   <property name="verticalSpacing" >
    <number>0</number>
   </property>
   <item row="1" column="0" colspan="2" >
    <widget class="QFrame" name="backgroundMain" >
     <property name="minimumSize" >
      <size>
       <width>760</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>760</width>
       <height>2048</height>
      </size>
     </property>
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>206</red>
           <green>207</green>
           <blue>210</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>206</red>
           <green>207</green>
           <blue>210</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>206</red>
           <green>207</green>
           <blue>210</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>206</red>
           <green>207</green>
           <blue>210</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font" >
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="autoFillBackground" >
      <bool>true</bool>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>10</number>
      </property>
      <property name="leftMargin" >
       <number>10</number>
      </property>
      <property name="topMargin" >
       <number>10</number>
      </property>
      <property name="rightMargin" >
       <number>10</number>
      </property>
      <property name="bottomMargin" >
       <number>10</number>
      </property>
      <item>
       <widget class="QTableWidget" name="dataTableCerts" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>740</width>
          <height>220</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>740</width>
          <height>2048</height>
         </size>
        </property>
        <property name="palette" >
         <palette>
          <active>
           <colorrole role="WindowText" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>127</red>
              <green>157</green>
              <blue>185</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>227</red>
              <green>226</green>
              <blue>210</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>127</red>
              <green>157</green>
              <blue>185</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>227</red>
              <green>226</green>
              <blue>210</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>119</red>
              <green>119</green>
              <blue>119</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>227</red>
              <green>226</green>
              <blue>210</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="autoFillBackground" >
         <bool>true</bool>
        </property>
        <property name="frameShape" >
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <property name="horizontalScrollBarPolicy" >
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="selectionBehavior" >
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled" >
         <bool>true</bool>
        </property>
        <row>
         <property name="text" >
          <string>1</string>
         </property>
        </row>
        <column>
         <property name="text" >
          <string>Issued To</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Issued By</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>CN</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Expiration</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Friendly Name</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>10</number>
        </property>
        <property name="leftMargin" >
         <number>10</number>
        </property>
        <property name="topMargin" >
         <number>10</number>
        </property>
        <property name="rightMargin" >
         <number>10</number>
        </property>
        <property name="bottomMargin" >
         <number>10</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonOK" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>75</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>75</width>
            <height>25</height>
           </size>
          </property>
          <property name="font" >
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text" >
           <string>OK</string>
          </property>
          <property name="shortcut" >
           <string>Ctrl+O</string>
          </property>
          <property name="default" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonCancel" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>75</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>75</width>
            <height>25</height>
           </size>
          </property>
          <property name="font" >
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text" >
           <string>Cancel</string>
          </property>
          <property name="shortcut" >
           <string>Ctrl+C</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonHelp" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>75</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>75</width>
            <height>25</height>
           </size>
          </property>
          <property name="font" >
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text" >
           <string>Help</string>
          </property>
          <property name="shortcut" >
           <string>Ctrl+H</string>
          </property>
          <property name="autoDefault" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>161</width>
            <height>25</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonImport" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>75</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>75</width>
            <height>25</height>
           </size>
          </property>
          <property name="text" >
           <string>Import...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLabel" name="uiBannerGraphicRight" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>450</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>450</width>
       <height>70</height>
      </size>
     </property>
     <property name="lineWidth" >
      <number>0</number>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap>images/banner_right.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="uiBannerGraphicLeft" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>310</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>310</width>
       <height>70</height>
      </size>
     </property>
     <property name="lineWidth" >
      <number>0</number>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap>images/banner_left.png</pixmap>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
